<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MMClassificiationDataAugmentationInput</class>
 <widget class="QWidget" name="MMClassificiationDataAugmentationInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupDataAugmentationTrainPipeline">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>541</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Train Pipeline</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout_17" rowstretch="1,0,0,0,0,0,0,0">
      <item row="1" column="3">
       <widget class="QSpinBox" name="spinBox_resizedCropSize">
        <property name="text" stdset="0">
         <string>224</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>224</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_randomErasing">
        <property name="text">
         <string>RandomErasing</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <layout class="QHBoxLayout" name="horizontalLayout_options">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <spacer name="horizontalSpacer_15">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="7" column="3">
       <layout class="QHBoxLayout" name="horizontalLayout_24"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_randomResiszedCrop">
        <property name="text">
         <string>Random Resized Crop Size</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_albuTransformType">
        <property name="text">
         <string>Albu Transform Type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="comboBox_albuTransformType">
        <property name="currentText">
         <string>ColorJitter</string>
        </property>
        <item>
         <property name="text">
          <string>ColorJitter</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="checkBox_randomErasing">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_albuTransformType">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_randomResizedCropSize">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_randomFlipProbability">
        <property name="text" stdset="0">
         <string>0,5000</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QComboBox" name="comboBox_randomFlipDirection">
        <property name="currentText">
         <string>horizontal</string>
        </property>
        <item>
         <property name="text">
          <string>horizontal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>vertical</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_randomFlipDirection">
        <property name="text">
         <string>Random Flip Direction</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_randomFlipProbability">
        <property name="text">
         <string>Random Flip Probability</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_randomFlipProbabilityAndDirection">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupDataAugmentationTestPipeline">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>541</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Test Pipeline</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout_18" rowstretch="1,0,0,0">
      <item row="1" column="3">
       <layout class="QHBoxLayout" name="horizontalLayout_TestOptions">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QSpinBox" name="spinBox_centerCropSize">
          <property name="text" stdset="0">
           <string>224</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="value">
           <number>224</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_resize">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_resize">
        <property name="text">
         <string>Resize</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="spinBox_resize">
        <property name="text" stdset="0">
         <string>256</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="value">
         <number>256</number>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <layout class="QHBoxLayout" name="horizontalLayout_25">
        <item>
         <spacer name="horizontalSpacer_16">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_centerCropSize">
        <property name="text">
         <string>Center Crop Size</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_centerCropSize">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>1</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_albuTransformType</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_albuTransformType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_randomResizedCropSize</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_resizedCropSize</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_randomFlipProbabilityAndDirection</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBox_randomFlipProbability</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_randomFlipProbabilityAndDirection</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_randomFlipDirection</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_resize</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_resize</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_centerCropSize</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_centerCropSize</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
